gateway/response/response.go

```diff
@@ -105,9 +105,9 @@ func GetStatusCode(key string) int {
 	if status, ok := statusMap[key]; ok {
 		return status.code
 	}
-	return http.StatusOK
+	return http.StatusInternalServerError
 }
 
 func MakeResponse(msg string) string {
-	return fmt.Sprintf(`{"msg": "%s"}`, msg)
+	return fmt.Sprintf("str: %q", msg)
```

Comment:
this doesn't look right & why would we want to change it?
```suggestion
	return fmt.Sprintf(`{"msg": %q}`, msg)
```

Reply:
Deepsource :)

Reply:
Oh Damn! I didn't realise it suggested the changed in the key. Will change it.